Src/Extern/SENSEI/AMReX_AmrParticleInSituBridge.H

```diff
@@ -0,0 +1,100 @@
+#ifndef AMReX_AmrParticleInSituBridge_H
+#define AMReX_AmrParticleInSituBridge_H
+#include <AMReX_Config.H>
+
+#ifdef AMREX_PARTICLES
+#include <AMReX_InSituBridge.H>
+#include <AMReX_Vector.H>
+
+#include <AMReX_AmrParticles.H>
+
+
+#include <AMReX_AmrParticleDataAdaptor.H>
+#include <AMReX_AmrDataAdaptor.H>
+
+#include <AMReX_Amr.H>
+
+#ifdef BL_USE_SENSEI_INSITU
+#include <AnalysisAdaptor.h>
+#include <Profiler.h>
+#endif
+
+#include <chrono>
+
+namespace amrex
+{
+/// Bridge for simulations that use amrex::ParticleContainer
+/// inherits intialize and finalize from amrex::InSituBridge
+template<int NStructReal, int NStructInt, int NArrayReal, int NArrayInt>
+class AmrParticleInSituBridge : public InSituBridge
```

Comment:
since there are no member variables that depend on the template parameters, the template should be on the update method not the class. this will make the class easier to use.